docs/sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,45 @@
 Version history
 ===============
 
+5.5.2 (2017 June 15)
+--------------------
+
+File format improvements:
```

Comment:
This section also lists bug fixes, perhaps it can be changed to "File format fixes and improvements"